src/pyedb/configuration/cfg_setup.py

```diff
@@ -177,6 +177,9 @@ def apply(self):
     def get_data_from_db(self):
         setups = []
         for _, s in self._pedb.setups.items():
+            if not s.type == "hfss":
+                # todo exporting siwave setup is skipped due to defect in 24.2
```

Comment:
Can you log a warning here to notify of such behavior ? Also, it might be a good idea to extend the if statement with a check on AEDT version, right ?

Reply:
Done